core/domain/exp_domain.py

```diff
@@ -776,6 +782,12 @@ def from_dict(
         exploration.created_on = exploration_created_on
         exploration.last_updated = exploration_last_updated
 
+        if exploration_dict.get('proto_size_in_bytes') is None:
```

Comment:
This seems off -- exploration_dicts should always come with all the necessary data, right? Or they shouldn't come with it, and this only gets added in the "domain object" formulation. But we should not have cases where sometimes the dict has the key-value pair and sometimes it doesn't.

Reply:
Fixed, and checked if any test fails without it or not. I did checked this part in early version of this PR, need to confirm again.

Reply:
Got it, why I added this check. 
There are places where we do 
`cls.from_dict(exploration_dict)` and the dict which we are passing is of the earlier version 54, and need to add proto_size_in_bytes which we added in 55 version. 

Will check this and fix it.

Reply:
I had updated the `from_yaml` function, as we are not keeping the proto_size_in_bytes in yaml content and removing it in `to_yaml` function, we need to add back in `from_yaml` function.

Reply:
Running all backend test locally and will push once all test pass.